.github/workflows/ci.yml

```diff
@@ -6,52 +6,20 @@ on:
     branches:
       - main
 
-env:
-  RUST_BACKTRACE: 1
-  CARGO_TERM_COLOR: always
-
```

Comment:
Any particular reason why we're removing these? They don't seem to be doing any harm and we want them around as a useful default in case other jobs get added in the future.